packages/experimental/Avatar/src/NativeAvatar.tsx

```diff
@@ -26,7 +26,8 @@ interface ExportedConstants {
   sizes: { [key in Size]: number };
 }
 
-const ExportedNativeConstants: ExportedConstants = TurboModuleRegistry.get('FRNAvatarConstants') || NativeModules.FRNAvatarViewManager;
+const ExportedNativeConstants: ExportedConstants =
+  TurboModuleRegistry.get('FRNAvatarConstants')?.getConstants() || NativeModules.FRNAvatarViewManager;
```

Comment:
I'll probably need to update the native constant-getting code here.

Reply:
(Or just remove it and move the constants to JS)

Reply:
Tested. Avatar just shows up as "Unimplemented Component".But also the size ramp seems to work so I think the constants are good. It doesn't crash so I'm happy. 

![image](https://github.com/microsoft/fluentui-react-native/assets/6722175/5e66a533-5114-4d68-bce6-66e49c399299)